pycontrails/core/flight.py

```diff
@@ -1112,6 +1113,101 @@ def filter_altitude(
         out.data.pop("level", None)  # avoid any ambiguity
         return out
 
+    def distance_to_coords(self: Flight, distance: ArrayOrFloat) -> tuple[
+        ArrayOrFloat,
+        ArrayOrFloat,
+        np.signedinteger[Any] | npt.NDArray[np.signedinteger[Any]],
+    ]:
+        """
+        Convert distance along fligth path to a geodesic coordinates.
```

Comment:
```suggestion
        Convert distance along flight path to geodesic coordinates.
```

Reply:
If I interpret the code right, this returns the geodesic coordinates for a distance as measured from the beginning of the flight? So from waypoint 0 -> some distance

And how would this expected to be used? Is there a simple example to add to the docstring? The output signature seems customized for something on the other end - is there an expectation about where/how this is used on output?

Can you update the docs (and changelog) to make this clear?